### PR TITLE
Update planning.md

### PR DESCRIPTION
**Pull request by dvhirst**
Reading for content, editing for consistency.

### CODE REVIEW

File: docs/basics/planning.md [hunk 1]
```diff
@@ -2,127 +2,61 @@
...
-**Avoid [hacking the core](/core/hacking.md)** of CiviCRM unless you understand
-the implications.
-
-**Follow the
-[Coding Standards](/standards/index.md)**
-for uniform structure that will make everyone's development work easier.
+**Open a free [GitHub](https://github.com/) account** for revision control. The official CiviCRM [repositories](https://github.com/civicrm) are housed in `git` repositories on GitHub.  If you use GitHub you will find it easy to access the latest source-code, to submit pull requests for any patches you create and to collaborate with many other CiviCRM developers who also use GitHub.
+
+**Install the [buildkit](https://github.com/civicrm/civicrm-buildkit)**, ideally as a [vagrant virtual-machine](https://github.com/civicrm/civicrm-buildkit-vagrant) or using one of available `docker` images ([1](https://github.com/progressivetech/docker-civicrm-buildkit), [2](https://github.com/ErichBSchulz/dcbk)). The buildkit is not an absolute
+requirement but it is definitely the fastest path to a good development experience!
```
Comment (seamuslee001) [inline docs/basics/planning.md]: @dvhirst should this be combined into the line above?

**Comment by seamuslee001** [comment]
This looks ok to me no content has been changed just re-organisation in the md file. I have added one comment which should be resolved before merging

File: docs/basics/planning.md (hunk 1 above)
Comment (dvhirst) [reply docs/basics/planning.md]: Missed combining line 46 into 45.  Oops.

Comment (dvhirst) [reply docs/basics/planning.md]: Missed combining line 79 / 46 into 78 / 45.  Oops.

**Comment by seamuslee001** [comment]
closing in favor of https://github.com/civicrm/civicrm-dev-docs/pull/361

**Pull request closed without merging**